server.js

```diff
@@ -1,54 +1,68 @@
-var util = require('util');
-var url = require('url');
-var qs = require('querystring');
-var os = require('os')
+#!/bin/env node
+//  OpenShift sample Node application
+var express = require('express');
+var fs      = require('fs');
+var app     = express();
+var eps     = require('ejs');
+
+app.engine('html', require('ejs').renderFile);
+
 var port = process.env.PORT || process.env.port || process.env.OPENSHIFT_NODEJS_PORT || 8080;
-var ip = process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0';
-var nodeEnv = process.env.NODE_ENV || 'unknown';
+var ip   = process.env.IP   || process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0';
+var mongoURL = process.env.OPENSHIFT_MONGODB_DB_URL || process.env.MONGO_URL;
```

Comment:
this needs to pick up the $DATABASE_SERVICE_NAME and use that instead of "MONGODB_*"

See how it's done in rails:
https://github.com/openshift/rails-ex/blob/master/config/database.yml#L3-L17